api/test/e2e/route_remote_addr_test.go

```diff
@@ -42,6 +42,7 @@ func TestRoute_add_with_invalid_remote_addr(t *testing.T) {
 				}`,
 			Headers:      map[string]string{"Authorization": token},
 			ExpectStatus: http.StatusBadRequest,
+			ExpectBody:   "scheme validate fail",
```

Comment:
`scheme` -> schema, `fail` -> `failed`
 Is there any specific failed reason?

Reply:
fixed.

Reply:
I have raised the issue of typo https://github.com/apache/apisix-dashboard/issues/914

Reply:
ping @moonming